tests/core/chain-object/test_chain.py

```diff
@@ -72,6 +76,52 @@ def test_empty_transaction_lookups(chain):
         chain.get_pending_transaction(b'\0' * 32)
 
 
+@pytest.mark.parametrize(
+    'data, gas_estimator, to, on_pending, expected',
```

Comment:
FYI, `pytest.mark.parametrize` can be stacked to produce a product of the two parameters

```
import pytest


@pytest.mark.parametrize('a', [1, 2])
@pytest.mark.parametrize('b', [3, 4])
def test_1(a, b):
    print(a, b)
```

This results in four test cases `[1, 3], [2, 3], [1, 4], [2, 4]`

Reply:
This is a good reminder, I had forgotten about that. In this case, I actually don't want to run all the permutations, because some of these tests are pretty slow. I just want a sampling of them. Running one test at 1000 tolerance takes 1-2 seconds, for example.

Reply:
Only to clarify my original comment:  I was suggesting it for the `on_pending` parameter which if I recall, each test was duplicated with this as `True/False`.  Don't care if you actually change anything here.